docs/resources/connector_internet.md

```diff
@@ -3,34 +3,42 @@
 page_title: "alkira_connector_internet Resource - terraform-provider-alkira"
 subcategory: ""
 description: |-
-  
+  Manage Internet Exit.
+  An internet exit is an exit from the CXP to theinternet and allows and allows the traffic fromthe various Users & Sites or Cloud Connectors toflow towards the Internet.
 ---
 
 # alkira_connector_internet (Resource)
 
+Manage Internet Exit.
 
+An internet exit is an exit from the CXP to theinternet and allows and allows the traffic fromthe various Users & Sites or Cloud Connectors toflow towards the Internet.
 
+## Example Usage
 
+```terraform
+resource "alkira_connector_inet" "test1" {
+  name           = "test1"
+  cxp            = "US-WEST"
+  group          = "group1"
+  segment        = "seg1"
+}
+```
```

Comment:
I think `size` is missing and seems to be a required attribute per the schema.

Reply:
Good catch!